mockgen/parse.go

```diff
@@ -61,6 +63,20 @@ func sourceMode(source string) (*model.Package, error) {
 		srcDir:             srcDir,
 	}
 
+	// positional interface names -> include set
+	ifaces := flag.Args()
+	if len(ifaces) > 0 {
+		p.includeNamesSet = make(map[string]struct{}, len(ifaces))
+		for _, arg := range ifaces {
+			for _, name := range strings.Split(arg, ",") {
+				name = strings.TrimSpace(name)
+				if name != "" {
+					p.includeNamesSet[name] = struct{}{}
+				}
+			}
+		}
+	}
```

Comment:
Other modes accept list of interfaces as a single argument, so I would recommend:

```suggestion
	if flag.NArg() > 1 {
		return nil, errors.New("-source mode accepts at most one argument")
	}
	if flag.NArg() == 1 {
		ifaces := strings.Split(flag.Arg(0), ",")
		p.includeNamesSet = make(map[string]struct{}, len(ifaces))
		for _, name := range ifaces {
			p.includeNamesSet[name] = struct{}{}
		}
	}
```